deb/openmediavault/lib/systemd/system-sleep/watchog

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
```

Comment:
Can you please rename this file to ``openmediavault-watchdog``, so it is clear that this file comes from OMV. Same applies to the pm-utils file.

Reply:
Ok, then let's call each script just `openmediavault` since maybe in the feature you might add more tweak that are not necessary related to watchdog. OK ?

Also I will add states hibernate+thaw to the script for pm-utilis since the same use case apply for hibernate state.